gen2-camera-demo/main.py

```diff
@@ -173,13 +175,16 @@ def build_pipeline(pipeline):
     #    stereo.setOutputRectified(out_rectified)
         #stereo.setBaselineOverrideCm(123.4)
         #stereo.setFovOverrideDegrees(1)
-        stereo.setConfidenceThreshold(200)
+        stereo.setConfidenceThreshold(100)
         stereo.setRectifyEdgeFillColor(0)  # Black, to better see the cutout
         stereo.setMedianFilter(median)  # KERNEL_7x7 default
         stereo.setLeftRightCheck(lrcheck)
         stereo.setExtendedDisparity(extended)
         stereo.setSubpixel(subpixel)
-        stereo.setDepthAlign(dai.StereoDepthProperties.DepthAlign.CENTER)
+        if enable_rgb:
+            stereo.setDepthAlign(dai.CameraBoardSocket.RGB)
+        else:
+            stereo.setDepthAlign(dai.StereoDepthProperties.DepthAlign.CENTER)
```

Comment:
Can you make else/default alignment to be in rectified left camera?  
on **WHY** : if the else statement is center aligned. The pcl projection when non-rgb is using rectified right camera frame and doesn't align well.
Maybe we can add another argument to move it to the center and have PCL projection for it without any stream overlap.

Thoughts?

Reply:
sounds good. Make default to `dai.StereoDepthProperties.DepthAlign.RECTIFIED_RIGHT` instead. 
Thanks a lot again for the additional changes.

Reply:
That is done. Note that I have defaulted to RGB when -rgb is enabled but --stereo-align isn't specifed and added a warning otherwise.

eg
```
Warning: --stereo_align should be set to rgb when -rgb is enabled.
```